modules/swagger-codegen/src/main/resources/cpprest/model-header.mustache

```diff
@@ -72,6 +86,6 @@ protected:
 }
 {{/modelNamespaceDeclarations}}
 
-#endif /* {{modelHeaderGuardPrefix}}_{{classname}}_H_ */
+#endif /* {{classname}}_H_ */
```

Comment:
Still wrong

Reply:
Whops, sorry, missed that one. Should be fixed now.